docs/helm/rbac-values.yaml

```diff
@@ -0,0 +1,10 @@
+# Contains account e-mails and namespaces as needed for the rolebinding templates.
+ClusterAdmins::
+ - YOUR_GCP_LOGIN_EMAIL
+
+#uses example namespaces
+Permissions:
+ - NamespaceName: "test"
```

Comment:
Just wondering, in this example only the `test` and `production` namespaces would be usable right? Or does this only restrict these namespaces to these service accounts but leave the others open for any service account?

Reply:
In this case, if we're deploying to that namespace, then those users have permissions. If users would like to add more users to particular namespaces, they can add some more -NamespaceName, GCPServiceAccount pairs.

Reply:
Believe users would be familiar with these concepts. Do you think we need to explain further?

Reply:
I guess I was more wondering, is this restricting access to a particular namespace to the given service account(s), or restricting all access to the cluster to these namespace/service account pairs?

Reply:
From: https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/

In summary, in the example, they had relaxed permissions for development and wanted to restrict that access to some users and likewise for production less permissive and restricted to certain users.

Reply:
Got it, and thanks for adding clarification to the comment there

Reply:
Ah, so to leave a more complete answer to your question, it won't restrict it to just those users. It just gives those users permission to that cluster with that namespace. If you want to add more users, you can do it anytime (though perhaps not with helm since you'll need to get rid of that release and install again).

Reply:
Thanks, that clears it up for me.